src/pages/PageFavorites.jsx

```
@@ -1,9 +1,47 @@
-      <h1>Favorites</h1>
+    <main className="flex flex-col items-center justify-center min-h-screen text-white text-center px-4">
+      <section className="max-w-xl mt-[-400px] sm:mt-0">
+        <h1 className="text-2xl md:text-4xl font-bold mb-4">
+          Oops! You haven't selected any favorites yet!
+        </h1>
+        <p className="text-base md:text-lg mb-2">
+          Browse or search for movies in the MovieFix Catalogue and start building your lists!
+        </p>
+        <br />
+        <p className="text-base md:text-lg mb-1">To add movies to your favorites list, tap on the heart icon.
+          <button class="cursor-pointer ml-2 mb-1 align-bottom">
```

ESLint

[error] Unknown property 'class' found, use 'className' instead

(react/no-unknown-property)